src/test/java/de/tum/in/www1/artemis/LearningGoalIntegrationTest.java

```diff
@@ -408,6 +436,70 @@ void deleteCourse_asAdmin_shouldAlsoDeleteLearningGoal() throws Exception {
         request.get("/api/courses/" + idOfCourse + "/goals/" + idOfLearningGoal, HttpStatus.NOT_FOUND, LearningGoal.class);
     }
 
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void createLearningGoalRelation() throws Exception {
+        Course course = courseRepository.findByIdElseThrow(idOfCourse);
+        Long idOfOtherLearningGoal = database.createLearningGoal(course).getId();
+
+        request.postWithoutResponseBody(
+                "/api/courses/" + idOfCourse + "/goals/" + idOfLearningGoal + "/relations/" + idOfOtherLearningGoal + "?type=" + LearningGoalRelation.RelationType.EXTENDS.name(),
+                HttpStatus.OK, new LinkedMultiValueMap<>());
+
+        var relations = learningGoalRelationRepository.findAllByLearningGoalId(idOfLearningGoal);
+        assertThat(relations).hasSize(1);
+        assertThat(relations.stream().findFirst().get().getType()).isEqualTo(LearningGoalRelation.RelationType.EXTENDS);
+    }
+
+    @Test
+    @WithMockUser(username = "student42", roles = "USER")
+    public void createLearningGoalRelation_shouldReturnForbidden() throws Exception {
+        Course course = courseRepository.findByIdElseThrow(idOfCourse);
+        Long idOfOtherLearningGoal = database.createLearningGoal(course).getId();
+
+        request.post(
+                "/api/courses/" + idOfCourse + "/goals/" + idOfLearningGoal + "/relations/" + idOfOtherLearningGoal + "?type=" + LearningGoalRelation.RelationType.EXTENDS.name(),
+                null, HttpStatus.FORBIDDEN);
+    }
+
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void gLearningGoalRelations() throws Exception {
```

Comment:
```suggestion
    public void getLearningGoalRelations() throws Exception {
```

I assume it should be like this